tests/Integration/IntegrationTest.php

```diff
@@ -64,9 +66,13 @@ public function testHtmlIntegration(
             system('mkdir ' . escapeshellarg($outputPath));
 
             $this->prepareContainer($configurationFile);
+
             $command = $this->getContainer()->get(Run::class);
+
             assert($command instanceof Run);
 
+            $command->setClock(new MockClock('2023-01-01 12:00:00'));
```

Comment:
You can set this in the `\phpDocumentor\Guides\DependencyInjection\TestExtension` then there is no reason the have the setClock method. You can register the MockClock there as a service with a certain argument.

Reply:
How could I set the mock time there?

Reply:
Ok, figured it out, setClock is removed